Makefile

```diff
@@ -19,7 +19,7 @@ include Makefile.common
 
 STATICCHECK_IGNORE =
 
-DOCKER_IMAGE_NAME ?= bind-exporter
```

Comment:
Oops, but good catch!

Reply:
In fact it was mentioned by @Tom-Fawcett in https://github.com/prometheus-community/prom-label-proxy/issues/41 (thanks!) but I missed the info...